amarna/command_line.py

```diff
@@ -142,7 +144,34 @@ def main() -> int:
         help="Disable rules with inline comments. The comments should be the first line and of the form: # amarna: disable=rulename1,rulename2",
     )
 
-    args = parser.parse_args()
+    parser.add_argument(
+        "-config",
+        "--config",
+        type=str,
+        help="Specify config file path.",
+    )
+    return parser.parse_args(argv)
+
+
+def get_config_from_file(config_file) -> Tuple[str, str, bool]:
+    if config_file is None:
+        print("No config file specified, trying with amarna.toml")
+        config_file = "amarna.toml"
+
+    try:
+        with open(config_file, "rb") as f:
+            config = tomli.load(f)
```

Comment:
Should check for decoding error when loading
https://github.com/hukkin/tomli#handle-invalid-toml